### DISCUSSION TITLE
OHS 12.2.1.4 - Image Size Big

### DISCUSSION THREAD

**Discussion by brunozanon10** [oracle/docker-images#2117] [General]
hello team,
I built the OHS using the procedure in the note: https://github.com/oracle/docker-images/tree/main/OracleHTTPServer/dockerfiles/12.2.1.3.0
Just swapping 12.2.1.3 for release 4, and the image got really big.
-------------------------------------------------------------------------------------
Successfully built 7a1a93a0ba02
Successfully tagged oracle/ohs:12.2.1.4.0

OHS Standalone Docker Image for version: 12.2.1.4.0 is ready to be used.
--> oracle/ohs:12.2.1.4.0
Build completed in 944 seconds.

$ docker images
REPOSITORY TAG IMAGE ID CREATED SIZE
oracle/ohs 12.2.1.4.0 7a1a93a0ba02 28 seconds ago 6.91GB
-------------------------------------------------------------------------------------

Does anyone know any way to optimize this size?

Thank you!
Bruno Zanon

**Comment by bamcgill** [comment]
what size was the 12.2.1.3 one?  Did you build on an existing one or create a fresh one.?

**Reply by brunozanon10** [reply]
> what size was the 12.2.1.3 one? Did you build on an existing one or create a fresh one.?

I didn't create 12.2.1.3, I replaced the scripts and binaries for release 12.2.1.4, and I created a new image. But the customer say it stayed too big.

**Comment by shindeswapnil55** [comment]
Hello Guys.. facing the same issue .. For me size is  6.65GB for OHS 12.2.1.4 image.. can anyone help on the same..
